apps/web-giddh/src/app/new-vs-old-Invoices/new-vs-old-Invoices.component.html

```diff
@@ -47,192 +47,560 @@
-            <div class="div-table bg-gray-color">
-                <div>
-                    <h3 class="h3-heading">{{bifurcationClients}} </h3>
-                </div>
-                <div class="div-row mr-t3 mr-b3">
-                    <div class="div-cell font-16 strong">{{localeData?.from_new_clients}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.newSales.uniqueCount | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.newSales.total | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.newSales.invoiceCount | giddhCurrency}}</div>
-                </div>
-                <div class="div-row mr-t3 mr-b3">
-                    <div class="div-cell font-16 strong">{{localeData?.from_old_clients}}</div>
-                    <div class="div-cell align-end">{{clientTotal | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{crdTotal | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{invTotal | giddhCurrency}}</div>
-                </div>
-                <div class="div-row">
-                    <div class="div-cell">{{(NewVsOldInvoicesData$ | async)?.carriedSales[0]?.month}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[0]?.uniqueCount | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[0]?.total | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[0]?.invoiceCount | giddhCurrency}}</div>
-                </div>
-                <div class="div-row">
-                    <div class="div-cell">{{(NewVsOldInvoicesData$ | async)?.carriedSales[1]?.month}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[1]?.uniqueCount | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[1]?.total | giddhCurrency}}</div>
-                    <div class="div-cell align-end">{{(NewVsOldInvoicesData$ | async)?.carriedSales[1]?.invoiceCount | giddhCurrency}}</div>
+        <ng-container *ngIf="!isLoading">
+                <div class="d-flex" *ngIf="selectedType==='month'">
+                    <div class="div-table">
+                        <div class="head-row">
+                            <div class="div-cell">{{localeData?.sales_from}}</div>
+                            <div class="div-cell align-end">{{localeData?.clients}}</div>
+                            <div class="div-cell align-end">
+                                {{commonLocaleData?.app_amount}}
+                                <span class="d-inline-flex">
+                                    <amount-field [amount]="commonLocaleData?.app_amount"
```

Comment:
this is text, don't use amount-field here

Reply:
Done